### ISSUE TITLE
ReferenceError: e is not defined

### ISSUE THREAD

**Issue by TheMrRoboto** [musiqpad/mqp-node-api#13]
I am getting a Reference Error as soon as I start the bot, its from the API files.

Here is my log:

```
[15/4/2017 21:44:17] - info: LogLevel: info
[15/4/2017 21:44:17] - info: Connecting to pad
[15/4/2017 21:44:17] - info: Connecting to socket
/home/mqp-bot/node_modules/mqp-api/src/app.js:159
      logger.log('error', "Websocket error: " + e);
                                                ^

ReferenceError: e is not defined
    at WebSocket._this.ws.onerror (/home/mqp-bot/node_modules/mqp-api/src/app.js:159:49)
    at WebSocket.onError (/home/mqp-bot/node_modules/ws/lib/WebSocket.js:452:14)
    at emitOne (events.js:96:13)
    at WebSocket.emit (events.js:191:7)
    at ClientRequest.onerror (/home/mqp-bot/node_modules/ws/lib/WebSocket.js:711:10)
    at emitOne (events.js:96:13)
    at ClientRequest.emit (events.js:191:7)
    at TLSSocket.socketErrorListener (_http_client.js:358:9)
    at emitOne (events.js:96:13)
    at TLSSocket.emit (events.js:191:7)
    at emitErrorNT (net.js:1283:8)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

**Comment by explodingcamera** [comment]
Can you send your bot config? There is probably an issue connecting to your websocket server.
It's weird that the error isn't defined though...

**Comment by TheMrRoboto** [comment]
What config would that be? The config.json I made for the bot?

**Comment by explodingcamera** [comment]
The object you pass to the bot API when initializing it.

**Comment by TheMrRoboto** [comment]
```javascript
var bot = new mqpAPI({
  autoreconnect: true,
  logging: {
    logLevel: 'info',
    logFile: './logs.json'
  }
});

bot.connect({
  useSSL: true,
  socketDomain: 'music.tfs-network.com',
  socketPort: 8002
}).then(() => {
  console.log('Connected!');
})
```

**Comment by explodingcamera** [comment]
fixed in c2189b657b86cc19879abaa24e7a9d6ca4344d50
